### PR TITLE
update boilerplate file to be KO Build Authors

### PR DESCRIPTION
**Pull request by cpanato**
Fixes: https://github.com/ko-build/ko/issues/1040

### CODE REVIEW

**Review by developer-guy — DISMISSED** [review]

File: hack/boilerplate/boilerplate.sh.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-# Copyright 2020 Google LLC All Rights Reserved.
+# Copyright 2020 KO Build Authors All Rights Reserved.
```
Comment (imjasonh) [inline hack/boilerplate/boilerplate.sh.txt:3]:
```suggestion
# Copyright 2020 ko Build Authors All Rights Reserved.
```

It's a silly styling thing, but usually it's lowercase.

(here and in boilerplate.go.txt)

Comment (cpanato) [reply hack/boilerplate/boilerplate.sh.txt:3]: roger that

Comment (cpanato) [reply hack/boilerplate/boilerplate.sh.txt:3]: done PTAL

File: hack/boilerplate/boilerplate.go.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Google LLC All Rights Reserved.
+// Copyright 2021 ko Build Authors All Rights Reserved.
```
Comment (imjasonh) [inline hack/boilerplate/boilerplate.go.txt:1]: Sorry to keep moving the target. I think this should be `Copyright 20XX ko Authors All Rights Reserved`.

Comment (cpanato) [reply hack/boilerplate/boilerplate.go.txt:1]: no worries!

Comment (cpanato) [reply hack/boilerplate/boilerplate.go.txt]: cannot change that :/ the job check fails then

```
Error: [Go headers] reported by reviewdog 🐶
  missing boilerplate:
  
  Raw Output:
  main.go:1: missing boilerplate:
  // Copyright 20XX ko Build Authors All Rights Reserved.
  //
  // Licensed under the Apache License
```

**Review by imjasonh — APPROVED** [review]

**Pull request merged**